ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/hosts.yml

```diff
@@ -23,6 +23,7 @@ all:
             mgmt_interface_dualstack:
             default_overlay_protocol_cvx:
             default_overlay_protocol_her:
+            evpn_l2vlan_bundle_definition:
```

Comment:
Let's just call it `evpn_vlan_bundle` so it resembles the key we are testing.

Reply:
done